spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/query/QuerySpec.java

```diff
@@ -125,6 +133,12 @@ public Builder limit(Integer limit) {
             return this;
         }
 
+        public Builder range(Range range) {
+            offset(range.getFrom());
+            limit(range.getTo());
```

Comment:
Shouldn't this be something like this?
```suggestion
            limit(range.getTo() - range.getFrom());
```

If we say "Get me everything from number 33 to number 35" your approach would return the entries 33 to 68 or not?

Reply:
you are correct. i overlooked that, thanks.